kokoro/builds/linux/clang7_relwithdebinfo/continuous.cfg

```diff
@@ -0,0 +1,25 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+before_action {
+  fetch_keystore {
+    keystore_resource {
+      keystore_config_id: 74938
+      keyname: "orbitprofiler_artifactory_access_token"
+      key_type: RAW
+      backend_type: FASTCONFIGPUSH
+    }
+    keystore_resource {
+      keystore_config_id: 74938
+      keyname: "orbitprofiler_crashdump_collection_server"
+      key_type: RAW
+      backend_type: FASTCONFIGPUSH
+    }
```

Comment:
I think that it should be added for `release.cfg` and removed from here.

Reply:
Good point. I just copied that from `kokoro/gcp_ubuntu/continuous.cfg`, but it should probably also not be there.
The `release.cfg` should also not exist for `clang7_relwithdebinfo` (No Linux-UI release). I had this for testing and I'm going to remove it.

Reply:
done.

Reply:
Is the same applicable for `gcc9_relwithdebingo`?

Reply:
Yes. thanks.

Reply:
done.